src/PrestaShopBundle/Command/ListCommandsAndQueriesCommand.php

```diff
@@ -62,27 +68,135 @@ public function configure()
         $this
             ->setName('prestashop:list:commands-and-queries')
```

Comment:
We have a tool, DocToolBundle, which has a command that has the same purpose, made differently but with a little more detail than the basic one. What is the direction we are taking with the modification of this one ? should we keep both ? https://github.com/PrestaShop/DocToolsBundle#command-list

Reply:
if so, you should see to remove it